csharp/src/Ice/Communicator-Properties.cs

```diff
@@ -235,8 +235,10 @@ internal PropertyValue(string v, bool u)
             }
         }
 
-        /// <summary>Gets the value of a property as a proxy. If the property is not set, returns null.</summary>
-        /// <param name="name">The property name. The property name is also used as the prefix for proxy options.</param>
+        /// <summary>Gets the value of a property as a proxy. If the property is not set, returns null. For ice2
+        /// proxies, this method is equivalent to parsing the property's value.</summary>
+        /// <param name="name">The property name. For ice1 proxies, the property name is also used as the prefix for
```

Comment:
I would document the ice1 behavior in the summary.

Reply:
Fixed.